rust/stackablectl/src/cmds/operator.rs

```diff
@@ -524,16 +530,15 @@ where
         Some(entries) => {
             let mut versions = entries
                 .iter()
-                .map(|e| Version::parse(&e.version))
-                .map_while(|r| match r {
-                    Ok(v) => Some(v),
-                    Err(_) => None,
+                .map(|entry| {
+                    Version::parse(&entry.version).context(InvalidHelmChartVersionSnafu {
```

Comment:
note: We could use `with_context` to only clone the string when an error occurs.

```suggestion
                    Version::parse(&entry.version).with_context(|_| InvalidHelmChartVersionSnafu {
```